src/Core/Exception/ErrorMessageException.php

```diff
@@ -29,9 +29,9 @@
 use Throwable;
 
 /**
- * Class PrestaShopCoreException.
+ * Class ErrorMessageException.
  */
-class PrestaShopCoreException extends \Exception
+class ErrorMessageException extends CoreException
```

Comment:
Isn't it rather a `TranslationAwareException` as suggested by @sarjon in https://github.com/PrestaShop/PrestaShop/pull/10865#issuecomment-427034772 ?

Reply:
yep good idea, although I don't like `TranslationAwareException` because it gives the feeling that the Exception is capable of translating itself, as if it contained the translator service
I prefered `TranslatableCoreException` which seems more correct to me